media-video/vlc/vlc-3.0.21_p20250204.ebuild

```diff
@@ -3,37 +3,22 @@
 
 EAPI=8
 
+COMMIT=3e64644242783c68892262b11c178a1c668ab14a
 LUA_COMPAT=( lua5-{1..2} )
-
-MY_PV="${PV/_/-}"
```

Comment:
Losing the template and ability to sync with 3.0.9999 is unfortunate, I think.

Reply:
otoh the other way would require adding the snapshot SRC_URI and bootstrap to 3.0.9999, for a hopefully only shortlived situation.